java/src/apps/ActionListBundle_fr.properties

```diff
@@ -61,9 +61,11 @@ jmri.jmrit.operations.locations.LocationsTableAction = Ouvrir Fen\u00eatre Explo
 jmri.jmrit.operations.rollingstock.cars.CarsTableAction = Ouvrir Fen\u00eatre Exploitation Wagons
 jmri.jmrit.operations.rollingstock.engines.EnginesTableAction = Ouvrir Fen\u00eatre Exploitation Machines
 jmri.jmrit.operations.routes.RoutesTableAction = Ouvrir Fen\u00eatre Exploitation Itin\u00e9raires
-jmri.jmrit.operations.trains.TrainsTableAction = Ouvrir Fen\u00eatre Exploitation Trains
+jmrit.operations.trains.TrainsTableAction = Ouvrir Fen\u00eatre Exploitation Trains
```

Comment:
Was this line meant to be changed this way? I Think the leading "jmri." needs to be retained for this action to be translated.